docs/concepts/what-is-a-temporal-cron-job.md

```diff
@@ -90,9 +90,11 @@ Consider that using time zones in production introduces a surprising amount of c
+  - For example, in the US, DST is at the 1AM hour. When you "fall back", the clock will go `1:59 … 1:00 … 1:01 … 1:59 … 2:00 … 2:01 AM` and any Cron jobs that falls in that 1AM hour will be fired again. The inverse happens when clocks "spring forward" for DST, and Cron jobs that fall in the 1AM hour are skipped.
+  - In other timezones like Chile and Iran, DST "spring forward" is at midnight. [11:59PM is followed by 1:00AM, which means `00:00:00` never happens](https://twitter.com/raphaelschaad/status/1425249265643839490?s=20&t=vrMx4ixqcBRp5D7xB4FgNw).
 - **Self Hosting note**: If you manage your own Temporal Cluster, you are responsible for ensuring that it has access to current `tzdata` files. The official Docker images are built with [tzdata](https://docs.w3cub.com/go/time/tzdata/index) installed (provided by Alpine Linux), but ultimately you should be aware of how tzdata is deployed and updated in your infrastucture.
 - **Updating Temporal**: If you use the official Docker images, note that an upgrade of the Temporal Cluster may include an update to the tzdata files, which may change the meaning of your Cron Schedule. You should be aware of upcoming changes to the definitions of the time zones you use, particularly around daylight saving time start/end dates.
-- **Absolute Time Fixed at Start**: The absolute start time of the next Run is computed and stored in the database when the previous Run completes, and is not recomputed. This means that if you have a Cron Schedule that runs very infrequently, and the definition of the time zone changes in between one Run and the next, the Run might happen at the wrong time. For example, `CRON_TZ=America/Los_Angeles 0 12 11 11 *` means "noon in Los Angeles on November 11" (normally not in DST). If at some point the government makes any chances (e.g. move the end of DST one week later, or stay on permanent DST year-round) the meaning of that specification will change. In that first year, the Run will happen at the wrong time, because it was computed using the older definition.
+- **Absolute Time Fixed at Start**: The absolute start time of the next Run is computed and stored in the database when the previous Run completes, and is not recomputed. This means that if you have a Cron Schedule that runs very infrequently, and the definition of the time zone changes in between one Run and the next, the Run might happen at the wrong time. For example, `CRON_TZ=America/Los_Angeles 0 12 11 11 *` means "noon in Los Angeles on November 11" (normally not in DST). If at some point the government makes any changes (e.g. move the end of DST one week later, or stay on permanent DST year-round) the meaning of that specification will change. In that first year, the Run will happen at the wrong time, because it was computed using the older definition.
```

Comment:
```suggestion
- **Absolute Time Fixed at Start**: The absolute start time of the next Run is computed and stored in the database when the previous Run completes, and is not recomputed. This means that if you have a Cron Schedule that runs very infrequently, and the definition of the time zone changes between one Run and the next, the Run might happen at the wrong time. For example, `CRON_TZ=America/Los_Angeles 0 12 11 11 *` means "noon in Los Angeles on November 11" (normally not in DST). If at some point the government makes any changes (for example, move the end of DST one week later, or stay on permanent DST year-round), the meaning of that specification changes. In that first year, the Run happens at the wrong time, because it was computed using the older definition.
```